src/pages/[platform]/deploy-and-host/fullstack-branching/secrets-and-vars/index.mdx

```diff
@@ -126,9 +126,9 @@ You can enable access to environment variables for your fullstack branch deploym
 
 You can manage your branch environment access through the Amplify console.
 
-1. First, create an environment variable in the Amplify console and name it REACT_APP_TEST_VARIABLE.
+1. First, create an environment variable in the Amplify console (in this example, you will name it `REACT_APP_TEST_VARIABLE`)
```

Comment:
bit out of scope for this PR but we should look into revising this section since `REACT_APP` env vars are specific to create-react-app